pages/common/assimp.md

```diff
@@ -0,0 +1,28 @@
+# assimp
+
+> Command-line client for the Open Asset Import Library.
+> Supports loading of 40+ 3D file formats, and exporting to several popular 3D formats.
+
+- List all supported import formats:
+
+`assimp listext`
+
+- List all supported export formats:
+
+`assimp listexport`
+
+- Convert a file to one of the supported output formats (see dox_cmd.h for a list of all common parameters):
```

Comment:
Not necessarily, but at least that's something googleable which should point them to the [dox_cmd.h](https://github.com/assimp/assimp/blob/master/doc/dox_cmd.h) file. Ideally, the rendered version (Doxygen output) should be available somewhere, but after a lot of searching, I could only find [this old archived version](https://laboratory.systems/p/apoc3d/source/file/1291/trunk/extra/assimp/doc/assimpcmd_html/common.html) (which doesn't even open in the browser, it triggers a download of the html page). Something closer to [this](http://htmlpreview.github.io/?https://github.com/ElviraBeck/GeKo/blob/master/dependencies/assimp/doc/assimpcmd_html/index.html) (with updated content) should be available either in http://assimp.sourceforge.net/lib_html/ or in https://assimp.github.io/assimp/ (or both!) but since it isn't, I figured at least pointing people to the source code file used to generate the documentation could be helpful, considering that it is easier to find than "the assimp_cmd docs" which the sub-commands' help mentions.

/cc @kimkulling